common/static/js/src/lang_edx.js

```diff
@@ -4,7 +4,7 @@ var Language = (function() {
         self = null;
     return {
         init: function() {
-            $settings_language_selector = $('#settings-language-value');
+            $settings_language_selector = $('#settings-language-value, input[name="radio-select-language"]');
```

Comment:
@ahmed-faraz46 Do we need to log this change

Reply:
Good point, we should create a wiki of core changes and manage it religiously for this project. I'll add one.